### PR TITLE
Cause kaleido to explicitly fail if no chromium:

### PR DESCRIPTION
**Pull request by ayjayt**
Before we were letting choreographer handle it.

### CODE REVIEW

**Comment by ayjayt** [comment]
Hey Greg, this error will be more clean for users that they need to have chrome installed!

**Comment by ayjayt** [comment]
Not sure if it solves https://github.com/plotly/Kaleido/issues/223 but it may

File: src/py/kaleido/scopes/plotly.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ def __init__(self, plotlyjs=None, mathjax=None, topojson=None, mapbox_access_tok
             'kaleido_scopes.js'
             )
         path = os.environ.get("BROWSER_PATH", which_browser())
+        if not path:
+            raise RuntimeError("Kaleido now requires that chrome/chromium is installed separately. Kaleido will try to detect it automatically, but the environmental error \"BROWSER_PATH\" can also be set")
```
Comment (johannesloibl) [inline src/py/kaleido/scopes/plotly.py:63]:
```suggestion
            raise RuntimeError("Kaleido now requires that chrome/chromium is installed separately. Kaleido will try to detect it automatically, but the environment variable \"BROWSER_PATH\" can also be set")
```

**Review by gvwilson — APPROVED** [review]

**Comment by pdmkdz** [comment]
While running jobs in Azure Databricks if Kaleido is installed during the job run, any plotly fig.show() will show the error message just added by this PR.

The only way to circumvent this is to set the rendering like this:
```
fig.show(renderer='databricks')
```

Again this error will not show if the notebooks in Azure databricks are run automatically, and may be due to how enviroment is handled by Databricks and not how Kaleido interacts with plotly.
Dropping this comment here because I know someone will pop their head soon...

**Comment by pdmkdz** [comment]
Not sure how to circumvent this new requirement in a serverless compute on Azure Databricks...
It seems chromium cannot be access in those conditions and will always end up in error if trying to save plotly plots as static images.

File: src/py/kaleido/scopes/plotly.py (hunk 1 above)
Comment (ayjayt) [reply src/py/kaleido/scopes/plotly.py:63]: wow okay this is correct why did I miss this? thank you

**Pull request merged**